src/Google.Api.Gax/ChannelPool.cs

```diff
@@ -32,11 +32,50 @@ public sealed class ChannelPool
             = new Lazy<Task<ChannelCredentials>>(
                 () => Task.Run(async () => (await GoogleCredential.GetApplicationDefaultAsync()).ToChannelCredentials()));
 
+        private readonly IEnumerable<string> _scopes;
+        private readonly Lazy<Task<ChannelCredentials>> _lazyScopedDefaultChannelCredentials;
```

Comment:
Nothing appears to use this.
Shouldn't GetChannel() be changed to use this as it's credential, rather than s_lazyDefaultChannelCredentials?
And can the non-scoped credentials be removed?